openedx/core/djangoapps/config_model_utils/models.py

```diff
@@ -37,27 +38,29 @@ class Provenance(Enum):
 
 class StackedConfigurationModel(ConfigurationModel):
     """
-    A ConfigurationModel that stacks Global, Site, Org, and Course level
+    A ConfigurationModel that stacks Global, Site, Org, Course, and Course Run level
     configuration values.
     """
     class Meta(object):
         abstract = True
 
-    KEY_FIELDS = ('site', 'org', 'course')
+    KEY_FIELDS = ('site', 'org', 'org_course', 'course')
```

Comment:
could we call these 'course' and 'course_run'?  
I'm assuming org_course is a course and course is a course_run?  
if so the current naming may be confusing

Reply:
so, the `org_course` is going to be specified as a string that includes both the org and the course. `HarvardX/CS50`, so that we don't get mixups if there are multiple orgs with the same course identifier. We could more aggressively name things as `course_run`, except I don't want to rename the column.

Reply:
the org there is just part of the course key field  
if we don't want to rename it perhaps we could at least update the docs to clarify that org_course should be populated with course key field, not course run key field and that course should be populated with course run key field?

Reply:
We should definitely document the expected formats. That said, in the context of the LMS, due to historical badness, `CourseKey` means "the key format that identifies a course run", and there is no standard term for "the key format that identifies as Course that contains runs".